public-ui/src/app/data-browser/charts/chart-biosex/chart-biosex-react.component.tsx

```diff
@@ -72,9 +72,11 @@ export class BioSexChartReactComponent extends React.Component<Props, State> {
             formatter: function () {
                    const label = this.axis.defaultLabelFormatter.call(this);
                    // Change <= 20 count to display '<= 20'
-                   if (label <= 20) {
-                    return '&#8804; 20';
-                   }
+                   if (series[0].dataOnlyLT20) {
+                    if (label <= 20) {
+                        return '&#8804; 20';
+                    }
+                    }
                    return label;
```

Comment:
Can simplify to a single if statement:
```suggestion
                   if (series[0].dataOnlyLT20 && label <= 20) {
                      return '&#8804; 20';
                    }
                   return label;
```
or even a one-liner:
```suggestion
                   return series[0].dataOnlyLT20 && label <= 20 ? '&#8804; 20' : label;
```

Reply:
Fixed thanks.